eng/native/gen-buildsys.sh

```diff
@@ -99,16 +99,7 @@ if [[ "$host_arch" == "wasm" ]]; then
     if [[ "$target_os" == "browser" ]]; then
         cmake_command="emcmake $cmake_command"
     elif [[ "$target_os" == "wasi" ]]; then
-<<<<<<< HEAD
-        if [[ -z $WASI_SDK_PATH ]]; then
```

Comment:
This check has no value anymore?

Reply:
It's more that it isn't in the upstream file (it is a goal to have little divergence). I will bring it back.